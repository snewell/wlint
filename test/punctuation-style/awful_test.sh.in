‘‘How are you?’’ Bob asked.

“As Frank said, ‘I’m okay,’” Lisa replied.

“ ‘You should be nicer,’ is what he always told me.”

It’s good to write clearly--unless it’s for effect.
