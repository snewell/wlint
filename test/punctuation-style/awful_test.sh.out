<stdin>-1:0 quotation.consecutive-opening-quotes
<stdin>-3:26 quotation.missing-space-closing-single-double
<stdin>-5:0 quotation.incorrect-space-opening-double-single
<stdin>-7:26 emdash.replace-double-hyphen
